BackendApartmentReservation/BackendApartmentReservation/Migrations/20190509095059_FullCarRent.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BackendApartmentReservation.Migrations
+{
+    public partial class FullCarRent : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
```

Comment:
Remove everything that is related to the `Groups`. There is something messed up :D I have fixed this in my PR